docs/modules/reference/pages/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -282,30 +282,67 @@ Note that you must include the `monitor` section in your definition.
-          <sendmail-host host="$\{ipaddr}" port="25" />
-          <sendmail-protocol mailer="smtpsend" />
-          <sendmail-message to="opennms@gmail.com" subject="OpenNMS Test Message"
-            body="This is an OpenNMS test message." />
-          <user-auth user-name="opennms" password="roolz" />
+        <sendmail-test attempt-interval="30000" <1>
+                       use-authentication="false" <2>
+                       use-jmta="false" <3>
+                       debug="false" > <4>
+          <sendmail-host host="$\{ipaddr}" <5>
+                         port="25" /> <6>
+          <sendmail-protocol mailer="smtpsend" /> <7>
+          <sendmail-message to=opennms@gmail.com  <8>
+                            subject="OpenNMS Test Message" <9>
+                            body="This is an OpenNMS test message." /> <10>
+          <user-auth user-name="opennms" <11>
+                     password="roolz" /> <12>
         </sendmail-test>
-        <readmail-test attempt-interval="5000" subject-match="OpenNMS Test Message" mail-folder="OPENNMS" debug="false" >
-          <readmail-host host="imap.gmail.com" port="993">
-            <readmail-protocol ssl-enable="true" start-tls="false" transport="imaps" />
+        <readmail-test attempt-interval="5000" <13>
+                       subject-match="OpenNMS Test Message" <14>
+                       mail-folder="OPENNMS" <15>
+                       debug="false" > <16>
+          <readmail-host host="imap.gmail.com" <17>
+                         port="993"> <18>
+            <readmail-protocol ssl-enable="true" <19>
+                               start-tls="false" <20>
+                               transport="imaps" /> <21>
           </readmail-host>
-          <user-auth user-name="opennms@gmail.com" password="opennms"/>
+          <user-auth user-name=opennms@gmail.com <11>
+                     password="opennms"/> <12>
         </readmail-test>
       </mail-test>
     </mail-transport-test>
   </parameter>
-  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-  <parameter key="ds-name" value="mtm_lat"/>
-  <parameter key="retry" value="20" />
+  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <22>
+  <parameter key="ds-name" value="mtm_lat"/> <23>
+  <parameter key="retry" value="20" /> <24>
 </service>
 
-<monitor service="MTM" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <1>
+<monitor service="MTM" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <25>
 ----
-<1> Required *monitor* section.
+<1> Interval in milliseconds between send attempts.
+<2> Whether to use authentication, in the event it is required.
+<3> Use the JavaMail Mail Transport Agent.
+<4> Show additional debug output.
+<5> The SMTP server address for sending mail.
+<6> The SMTP server port.
+<7> Use smtpsend or an alternate mailer.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.